popup/folderlist.js

```diff
@@ -429,7 +434,13 @@ class TBFolderList extends HTMLElement {
     }
   }
 
-  repopulate() {
+  async repopulate() {
+    let vars = await browser.storage.local.get({ showFolderPath: true });
```

Comment:
The folder list is meant to be a binding/widget that is independent of the WebExtensions API. I think what we'll need to do here is move getting the preference into popup.js, making showFolderPath a property on the binding, and having a storage.onChanged listener to update the showFolderPath property as necessary.

Otherwise this looks great! I realize I've been neglecting the PRs on this repo, if you are not up for making these changes I'll try to get it done myself soon. I do appreciate your contribution!

Reply:
@kewisch Hi Philipp, thanks for your feedback and interest in this change.

I made the requested modification so "folderlist" is independent of the WebExtensions API ; let me know if you would like to see any additional change.